fixtures/generate-fixtures.sh

```diff
@@ -7,7 +7,7 @@
 # POSTGRES_USER=atlasprd3 \
 # POSTGRES_DB=gxpscxadev \
 # generate-fixtures.sh \
-# E-CURD-4 E-EHCA-2 E-GEOD-71585 E-GEOD-81547 E-GEOD-99058 E-MTAB-5061
+# 'E-CURD-4 17' 'E-EHCA-2 24' 'E-GEOD-71585 19' 'E-GEOD-81547 24' 'E-GEOD-99058 7' 'E-MTAB-5061 25' 'E-ENAD-53 14'
```

Comment:
Above experiment, accessions are a bit confusing, why we added -  17, 24 19..etc to the existing experiment accessions.

Reply:
+1

Reply:
Alfonso, could you please clarify the above changes, please?

Reply:
It’s now explained in the README file of the fixtures directory.